### ISSUE TITLE
Daily schedule regeneration doesn't wipe table before adding the next day

### ISSUE THREAD

**Issue by MoeFwacky** [justinemter/pseudo-channel#31]
When the script generated the next day's schedule just after midnight, it didn't wipe the previous day's schedule from the table causing entries for both days to appear in the schedule.

If I stop and re-run the script with -xml -g included, it wipes the previous schedule and generates a new one as expected. The bug only occurs with the automatic daily refresh.

**Comment by justinemter** [comment]
Hmm, not sure what happened there. I had an sql statement that drops all data from the "daily_schedule" table right before it adds the new schedule "WHERE id > -1". I figure that last condition is probably not needed so removed it and pushed to develop. If it happens again then I will assume that the app is trying to do too much at once and need to move that statement to the top of the generate_daily_schedule function. I have a feeling that the new update will fix it though. :)

**Comment by justinemter** [comment]
Also, the `-i` flag doesn't do anything now. It was used when I wrote the commercial injection algorithm and I've since kept the option there to perhaps add manual injection. Thus far it does not seem like this would add any benefit so I've left the function empty. I'd stick to using only the flags outlined in the README.md of the 'develop' branch.

**Comment by MoeFwacky** [comment]
I updated when I got home and I'm running with `-xml -r`. Will it still generate the daily schedule without `-g`? I didn't run `-g` because I didn't want to overwrite today's schedule and advance all of the episodes.

**Comment by justinemter** [comment]
hmm, It should when the clock hits midnight. I made a minor change to the sql query. The app will run `-g` at midnight as usual but will trigger a slightly different sql command. If it still doesn't work I will try and approach the problem as if it's a bottleneck situation and move that function where there is less traffic. You don't need to change a thing outside of updating the repo and ensuring you're in the develop branch. I just pushed an update to export the queue via json via the `-e` flag. Be careful though because if you accidentally run `python PseudoChannel.py -i` and you have an older .json file that was exported it a few days ago it will overwrite your queue with your older export. I'll add a Y/N confirmation eventually but this should be nice for testing / debugging / enjoying TV in the mean time. I'll hit you discord tomorrow.

**Comment by justinemter** [comment]
Oh, I also added the bash scripts / real time playhead / queueing thing, you are probably referring to. It shouldn't affect anything (fingers crossed). When you run `python PseudoChannel.py -r` it should run as per usual but also queue up any media that is currently playing seeking to the correct position. The bash scripts I added today are for interacting with the app externally using something like a clap on / clap off thing or whatever. I'd imagine the most usable use-case is setting up an IR receiver to your raspberry pi to listen for a button that isn't used often, and trigger `./startstop.sh` when it "hears" it. Anyway, disregard that for now as we make sure everything else is settled.
